app/src/main/java/com/codepath/chefster/activities/DishDetailsActivity.java

```diff
@@ -0,0 +1,100 @@
+package com.codepath.chefster.activities;
+
+import android.support.design.widget.TabLayout;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentPagerAdapter;
+import android.support.v4.view.ViewPager;
+import android.os.Bundle;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+
+import com.bumptech.glide.Glide;
+import com.codepath.chefster.R;
+import com.codepath.chefster.fragments.IngredientsFragment;
+import com.codepath.chefster.fragments.ReviewFragment;
+import com.codepath.chefster.models.Dish;
+
+import org.parceler.Parcels;
+
+import java.util.ArrayList;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+public class DishDetailsActivity extends BaseActivity {
+    @BindView(R.id.vpDish)
+    ViewPager viewPagerDish;
+
+    @BindView(R.id.tlDish)
+    TabLayout tabLayoutDish;
+
+    @BindView(R.id.ivDishDetails)
+    ImageView ivDishDetails;
+
+    final static private int FIRST = 0;
+    final static private int SECOND = 1;
+    private Dish dish;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_dish_details);
+
+        ButterKnife.bind(this);
+
+        dish = Parcels.unwrap(getIntent().getParcelableExtra("selected_dish"));
```

Comment:
It's not urgent but maybe we should add a `final static public String DISH_KEY = "selected_dish" to `ChefsterApplication.java` and then you can write `ChefsterApplication.DISH_KEY` and we can call it from all activities that need it. It will be good in case we would want to change it (for some reason) and then we need to change it only in one place.